sycl/source/detail/queue_impl.hpp

```diff
@@ -264,7 +264,16 @@ class queue_impl {
       destructorNotification();
 #endif
       throw_asynchronous();
-      getAdapter()->call<UrApiKind::urQueueRelease>(MQueues[0]);
+      auto status =
```

Comment:
l278, exception handler is unreachable now.
__SYCL_CHECK_UR_CODE_NO_EXC checks and prints only.